src/frontend/src/reducers/networks/utils/fetchAndProcessDataByDestination.ts

```diff
@@ -0,0 +1,111 @@
+// Refactored from old codebase loadDataset and fetchAllTimesAndPathsForCoordinate at commit 6f17e33
+// New source code copied from latest Taui v3
+// ATTR: Taui by Conveyal, included under the MIT license (https://github.com/conveyal/taui/blob/dev/LICENSE)
```

Comment:
This file is just the logic from getTimesAndPathsDataForPlace pulled out into a fn that can also be called from the bulk data fetch thunk